src/helper.js

```diff
@@ -52,9 +53,22 @@ const Helper = {
 	},
 };
 
-module.exports = Helper;
+function requireConfig(configPath_) {
```

Comment:
why `configPath_`? Can we not just use `configPath` again? If it's to signal that it's private I'm used to seeing it prefixed with _, like `_configPath`, and that's not generally consistent with our codebase for function params

Reply:
the linter complains about both `configPath` and `path` because they shadow global variables, and I couldn't think of a better name

Reply:
`filePath`?